dev-toolkit/docker-compose.consumer.yaml

```diff
@@ -1,5 +1,6 @@
 services:
   consumer-ok:
+    platform: linux/amd64
```

Comment:
why specific to linux/amd64?

Reply:
there is no arm image right now, and by default it will try to download the arm image on a m1 based mac

Reply:
I think consumer is a regular kafka or spring kafka application? If so I think an image multi arch can be provided

Reply:
Yes, of course it's possible. I have to change how github builds the image before.  We can "pause" this PR meanwhile